freqtrade/exchange/gateio.py

```diff
@@ -39,6 +42,11 @@ class Gateio(Exchange):
         (TradingMode.FUTURES, MarginMode.ISOLATED)
     ]
 
+    def _init_ccxt(self, exchange_config: Dict[str, Any], ccxt_module: CcxtModuleType = ccxt,
+                   ccxt_kwargs: Dict = {}) -> ccxt.Exchange:
+        ccxt_kwargs.update({"timeout": 15000})
+        return super()._init_ccxt(exchange_config, ccxt_module, ccxt_kwargs)
+
```

Comment:
I don't see why this would be relevant / related to this PR, please remove it.

Gateio's api performs quite well and fast in my tests - if you're having timeout problems, then it's most likely a network problem - not something we will enforce at a freqtrade level.
You can configure this in the settings - but clearly not here.

Reply:
https://github.com/freqtrade/freqtrade/runs/5818814987?check_suite_focus=true

Reply:
Most likely, this suggests that gate.io blocked the github actions IP range.
Increasing the timeout for all users will not be the solution.

Either way - it's not related to this PR - therefore please remove it.

Reply:
Happened in GCP too. Will fill an issue at `ccxt`.  Currently have no logs.